vmware_aria_operations_integration_sdk/api/integration-sdk-definition-endpoint.json

```diff
@@ -322,6 +322,9 @@
         "type": "object",
         "description": "Definition of the object model returned in CollectResult",
         "properties": {
+          "version":{
```

Comment:
"version" is ambiguous, especially since it's inside an 'AdapterDefinition' object. We should clarify that it is the version of the adapter definition schema, rather than the adapter definition itself (this is especially true because the describe.xml file is also versioned, but it _is_ the for versioning the adapter definition itself).